src/Thumbnail.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classSet from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+
+const Thumbnail = React.createClass({
+  mixins: [BootstrapMixin],
+
+  getDefaultProps() {
+    return {
+      bsClass: 'thumbnail'
+    };
+  },
+
+  render() {
+    let classes = this.getBsClassSet();
+
+    if(this.props.href) {
+      return (
+        <a {...this.props} href={this.props.href} className={classSet(this.props.className, classes)}>
```

Comment:
If we just warn, that's not very enabling.

I see two options:

1. Render the anchor tag around the image only.
2. Render the anchor tag around the image and it's children.

Option 1 is probably the safest since it's entirely possible to pass a button or anchor as a child. The question is though, what is the prescribed Bootstrap solution or is this one up to interpretation by us?